### PR TITLE
fix: markdown editor auto scrolls

### PR DESCRIPTION
**Pull request by nguyengiang2000**
What does it do?
This pull request fixes an issue where the Markdown Editor auto-scrolls to the top when applying formatting (e.g., bullet lists, headings). The editor's position is maintained during formatting operations, improving the editing experience.

Why is it needed?
The auto-scrolling issue disrupts the user's workflow by jumping to the top of the page when formatting is applied. This fix ensures that the editor maintains its position, allowing users to continue editing without unnecessary scrolling.

How to test it?
1.Open the Markdown Editor in the application.
2.Create or open a document with some content.
3.Apply different types of formatting (e.g., bullet lists, headings) to the content.
4.Verify that the editor does not scroll to the top during or after applying the formatting.

Related issue(s)/PR(s)
Fixes https://github.com/strapi/strapi/issues/20160.

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Comment by hanpaine** [comment]
Hey @nguyengiang2000 thanks very much for your PR! We will review it soon 🙏

**Comment by nguyengiang2000** [comment]
Thank you for your response.

**Comment by derrickmehaffy** [comment]
Hi @nguyengiang2000 it looks like there is a lot of conflicts here and I suspect this fix was originally based on a Strapi 4 branch rather than Strapi 5. Can you rebase your changes on the current development branch.

Thanks,

**Comment by nguyengiang2000** [comment]
Can you tell me more specificly what should i do ?

**Comment by Marc-Roig** [comment]
@nguyengiang2000 we modified the base branch (we moved the v4 code into "v4")
we will be reviewing this pr during this week/next one

**Comment by nguyengiang2000** [comment]
Awesome, thanks for the update!

File: .github/actions/check-pr-status/dist/index.js
Comment (remidej) [inline .github/actions/check-pr-status/dist/index.js:1]: I assume this diff comes from us juggling with different branches on this PR — could you revert it? There shouldn't be any change here

**Review by remidej — COMMENTED** [review]
Hello, sorry for the delay but I got finally got around to testing your fix, and it works nicely 👌 

There's just 1 change that breaks the CI, I can merge once you revert it

**Review by remidej — APPROVED** [review]

**Pull request merged**